backend/codesirius/problems/tests/test_tag_api.py

```
@@ -0,0 +1,169 @@
+        )
+
+        self.assertEqual(res.status_code, status.HTTP_401_UNAUTHORIZED)
+
+    def test_update_tag_requires_admin(self):
+        """Test that updating a tag requires admin privileges."""
+        self.client.force_authenticate(user=self.user)
+        tag = create_tag()
+        payload = {"name": "Updated Tag"}
+
+        res = self.client.patch(
+            reverse("tag-retrieve-update-destroy", args=[tag.id]), payload
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)